include/inviwo/core/ports/portinspectormanager.h

```diff
@@ -27,15 +27,16 @@
  *
  *********************************************************************************/
 
-#ifndef IVW_PORTINSPECTORMANAGER_H
-#define IVW_PORTINSPECTORMANAGER_H
+#pragma once
 
 #include <inviwo/core/common/inviwocoredefine.h>
-#include <inviwo/core/common/inviwo.h>
 #include <inviwo/core/network/processornetworkobserver.h>
 #include <inviwo/core/io/serialization/serializable.h>
+#include <inviwo/core/util/glmvec.h>
```

Comment:
possible to use #include <glm/fwd.hpp>?

Reply:
this not imported much does not matter so much...